http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/HttpRoutesSpec.scala

```diff
@@ -556,6 +558,41 @@ class HttpRoutesSpec
     }
   }
 
+  // Validate encoding/decoding of RuntimeConfigRequest types
```

Comment:
Added some tests that round-trip encode/decode these types.

I put it here because the `RuntimeConfigRequest.*` decoders are defined in the `http` module in `RuntimeRoutes`. I could see moving this to `JsonCodec`/`JsonCodecSpec` as well.